addons/hr/models/hr_employee.py

```diff
@@ -134,6 +134,56 @@ def _default_image(self):
         ('user_uniq', 'unique (user_id, company_id)', "A user cannot be linked to multiple employees in the same company.")
     ]
 
+    @api.multi
+    def name_get(self):
+        try:
```

Comment:
why not check access rights to make this one symmetric with the other methods below